.github/workflows/publish.yml

```diff
@@ -1,10 +1,8 @@
 name: Publish to PyPi
 
 on:
-  push:
-    tags:
-      - 'v*'
-  workflow_dispatch:
```

Comment:
this line is what allows manual publishing; we may want to keep it

Reply:
What is the purpose of manual publishing?

Reply:
Debugging. It would allow us to retry the workflow without making a new release, and while we can retry a failed release action, it can be handy for retrying when the action technically succeeds but didn't do what you want.